### PR TITLE
add: language jp support

### PR DESCRIPTION
**Pull request by JonasGao**

### CODE REVIEW

**Comment by axeker11** [comment]
pull request

**Comment by axeker11** [comment]
JonasGao referenced this pull request from another issue

**Comment by axeker11** [comment]
#1059

File: shadowsocks-csharp/Data/jp.txt [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+Proxy Port=代理ポート
+
+# Log Form
+
+&File=ファイル (&F)
+&Open Location=この場所を開く (&O)
+E&xit=終了 (&X)
+&View=ビュー
+
+(&V)
```
Comment (wongsyrone) [inline shadowsocks-csharp/Data/jp.txt]: &View这行咋回事

Comment (celeron533) [inline shadowsocks-csharp/Data/jp.txt]: 啊，那时候没注意到。。。
抱歉
image

**Pull request merged**